int32 CH1
int32 CH2
int32 CH3
int32 CH4
int32 CH5
int32 CH6
